Source/WebKit/Shared/Sandbox/macOS/common.sb

```diff
@@ -45,15 +45,8 @@
 (with-filter (mac-policy-name "Quarantine")
     (allow system-mac-syscall (mac-syscall-number 84 87)))
 
-(allow file-test-existence (with telemetry))
-(allow file-test-existence
-    (literal "/etc/localtime")
-    (literal "/private/etc/localtime")
-    (subpath "/System/Library/PrivateFrameworks")
-    (subpath "/System/Library/Frameworks")
-    (subpath "/usr/share/icu")
-    (subpath "/private/var/db/timezone")
-    (subpath "/var/db/timezone"))
+(allow file-test-existence)
```

Comment:
I think this is fine, but did you intend to allow global `file-test-existance`, and not just keep the ones in the list? I assume telemetry showed an unbounded set of files needed for this check?

Reply:
This is a good question!

There's no obvious security consequence to testing file existence, but I can imagine some potential consequences, so ideally we would limit to a set of paths or subpaths.